docs/conf.py

```diff
@@ -79,6 +80,16 @@
 random.seed(0)
 '''
 
+# This config value must be a dictionary of external sites, mapping unique
+# short alias names to a base URL and a prefix.
+# See http://sphinx-doc.org/ext/extlinks.html
+extlinks = {
+    'gh-link': ('https://github.com/HypothesisWorks/hypothesis-python/%s', ''),
```

Comment:
the smallest of nits: worth keeping the dict visually alphabetical by key?

I don't know if the dict will grow much, but it is useful to be able to run a visual bisecting search on sets of a certain size.